src/amo/components/ScreenShots.js

```diff
@@ -17,59 +28,103 @@ const PHOTO_SWIPE_OPTIONS = {
   counterEl: true,
   arrowEl: true,
   preloaderEl: true,
-  // Overload getThumbsBoundsFn as workaround to
+  // Overload getThumbBoundsFn as workaround to
   // https://github.com/minhtranite/react-photoswipe/issues/23
-  getThumbBoundsFn: /* istanbul ignore next */ function getThumbBoundsFn(index) {
-    const thumbnail = document.querySelectorAll('.pswp-thumbnails')[index];
+  getThumbBoundsFn: (index: number, {
+    _document = typeof document !== 'undefined' ? document : null,
```

Comment:
@kumar303 Flow complains about this line because the maybe/union type `typeof document | null` does not work. I don't know why and I am stuck right now.

Reply:
Looks like a Flow bug that hasn't been fixed: https://github.com/facebook/flow/issues/183